projects/helper/bitcoin-book/hashkey-exchange.js

```diff
@@ -4,4 +4,6 @@ module.exports = [
   "1NfJSiqBw4fb74KgVQrPsk5W5aqitAD1Xv",
   //    "bc1qlrawqecuwgpzzwh04pkhtfsqsk33kald22ds3c", // bosera funds https://www.bosera.com/english/index.html
   "18oxobhCNKnHjb7nEFDmPdXbCZthFWezrm",
+  "bc1q9d5lq9psmkx9rtgewjgez7csg45faak2cccew8", // 2025-05-26
+  "bc1qlrawqecuwgpzzwh04pkhtfsqsk33kald22ds3c", // 2025-05-26
```

Comment:
Please remove this wallet
This is the ETF part , not cold/hot wallets 
https://www.investing.com/etfs/9008
https://markets.ft.com/data/etfs/tearsheet/summary?s=3008:HKG.HS